doc/nrf/libraries/networking/lwm2m_location_assistance.rst

```diff
@@ -41,6 +41,74 @@ There are three different supported methods of obtaining the location assistance
 * Query of P-GPS predictions - The P-GPS predictions are queried from nRF Cloud and provided back to the device through the LwM2M server.
   The predictions are stored in the device flash and injected to the GNSS module when needed.
 
+API usage
+*********
+
+Cell based location
+===================
+
+Cell based location can be based only on current cell or current and neighboring cells.
+Getting the information about the current cell is achieved by using the Connectivity Monitor object (ID 4).
+By enabling the kconfig option :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_CONN_MON_OBJ_SUPPORT`, the connectivity monitor is populated with data about current cell.
+
+Getting the information about the neighboring cells is achieved by using a collection of ECID-Signal Measurement Information object (ID 10256).
+These are populated by calling :c:func:`lwm2m_ncell_handler_register` to register the listener for the neighborhood measurements and scheduling a measurement by calling :c:func:`lwm2m_ncell_schedule_measurement`.
+
+The Ground Fix Location object has only one resource that needs to be addressed before sending a location request, `report_back`.
+The back reporting tells the server if it needs to send the acquired location back to the device or not.
+If the location is sent back to the device, the location is stored only in the Ground Fix Location object.
+
+To finally send the location request for the cell based location, call :c:func:`location_assistance_ground_fix_request_send`.
+
+A-GPS location
+==============
+
+When using A-GPS location, the device requests A-GPS assistance data from the server.
+GNSS-module can be queried for the data it needs. Device can ask all the data in one go or split the request to reduce the memory usage.
+The request also contains information about the current cell the device is connected to and the information is similarly available on Connectivity Monitor object as in the cell based location.
+
+When requesting for A-GPS assistance data, the device should first set the mask for the data it is requesting by calling :c:func:`location_assistance_agps_set_mask`.
+After the mask has been set, :c:func:`location_assistance_agps_request_send` sends the request with all the necessary data to the server and responds with the A-GPS assistance data.
+The assistance data is written to the A-GPS module automatically by the library.
+
+Filtered A-GPS
+--------------
+
+There is also an option to use filtered A-GPS. In it, the satellites below the given angle above the ground will be filtered out.
+The angle can be set with :c:func:`location_assist_agps_set_elevation_mask` to a degree between `[0, 90]`. Setting the degree to -1 disables the filtering and is used by default.
```

Comment:
```suggestion
There is also an option to use filtered A-GPS.
With this option, the satellites below the given angle above the ground are be filtered out.
You can set the the angle to a degree `[0 - 90]` using the :c:func:`location_assist_agps_set_elevation_mask` function. Setting the degree to `-1` disables filtering.
This is the default setting.
```

Reply:
I think this feels clearer:
Setting the degree to `-1` disables filtering, which is the default setting.